pulp_ansible/tests/functional/api/collection/v3/test_client_configuration.py

```diff
@@ -0,0 +1,8 @@
+def test_client_default_base_path(
+    ansible_repo, ansible_distribution_factory, ansible_client_configuration_api_client
+):
+    distro = ansible_distribution_factory(ansible_repo)
+
+    res = ansible_client_configuration_api_client.get(path=distro.base_path)
```

Comment:
How about another get without the path argument?

Reply:
`ANSIBLE_DEFAULT_DISTRIBUTION_PATH` is none by default, and since I can't change the settings during tests, I don't think it's possible to the configuration setting for the default distribution

Reply:
What happens if we set it? Will all the tests break?

I'm currently pitching this to cover setting _and_ not setting feature flags in the ci:
https://github.com/pulp/plugin_template/pull/692